spec/concerns/queue_it/queable_spec.rb

```diff
@@ -138,23 +138,78 @@
         task.push_to_queue(third_nodable, in_head)
       end
 
-      it "" do
-        expect(task.get_next_in_queue.nodable).to eq(third_nodable)
+      it "expects to get third nodable and to correctly move the queue" do
+        expect(task.get_next_node_in_queue.nodable).to eq(third_nodable)
         expect(task.queue.nodes.find_by(kind: :head).nodable).to eq(second_nodable)
         expect(task.queue.nodes.find_by(kind: :any).nodable).to eq(first_nodable)
         expect(task.queue.nodes.find_by(kind: :tail).nodable).to eq(third_nodable)
       end
     end
+
+  end
+
+  describe '#formatted_queue' do
+    let(:action_to_call) { 'name' }
+
+    context "when queue dosen't exist or is empty" do
+      it { expect(task.formatted_queue(action_to_call)).to be(nil) }
+    end
+
+    context 'when queue has one node' do
+      let(:nodable) { create(:user) }
+
+      before { task.push_to_queue(nodable) }
+
+      it "expects to receive an array with nodable name" do
+        expect(task.formatted_queue(action_to_call)).to eq([nodable.name])
+      end
+    end
+
+    context 'when queue has two nodes' do
+      let(:first_nodable) { create(:user) }
+      let(:second_nodable) { create(:user) }
+      let(:in_head) { true }
+
+      before do
+        task.push_to_queue(first_nodable)
+        task.push_to_queue(second_nodable, in_head)
+      end
+
+      it "expects to receive array with second and first nodable's names" do
+        expect(
+          task.formatted_queue(action_to_call)
+        ).to eq([second_nodable.name, first_nodable.name])
+      end
+    end
+
+    context 'when queue is generic (has more than two nodes)' do
+      let(:first_nodable) { create(:user) }
+      let(:second_nodable) { create(:user) }
+      let(:third_nodable) { create(:user) }
+      let(:in_head) { true }
+
+      before do
+        task.push_to_queue(first_nodable)
+        task.push_to_queue(second_nodable, in_head)
+        task.push_to_queue(third_nodable, in_head)
+      end
+
+      it "expect to receive array with third, second and frist nodable's names" do
+        expect(
+          task.formatted_queue(action_to_call)
+        ).to eq([third_nodable.name, second_nodable.name, first_nodable.name])
+      end
+    end
   end
 
-  describe '#empty_queue' do
-    describe 'when queue is empty' do
+  describe '#delete_queue_nodes' do
+    context 'when queue is empty' do
       before { task.find_or_create_queue! }
 
-      it { expect { task.empty_queue }.not_to change { task.queue.size } }
+      it { expect { task.delete_queue_nodes }.not_to change { task.queue.size } }
```

Comment:
⚠️ **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Lint/AmbiguousBlockAssociation: Parenthesize the param change { task.queue.size } to make sure that the block will be associated with the change method call.